src/utils/searchUtils.ts

```diff
@@ -14,6 +14,77 @@ import { RESULTS_PER_PAGE } from "../config/constants"
 import { collapseMultiValueQueryParams } from "./refineSearchUtils"
 import { getPaginationOffsetStrings } from "./appUtils"
 
+export const searchFormOptions = [
```

Comment:
Should this go in constants.ts? I think we should keep mostly functions in util files for consistency.